### PR TITLE
Open attachment links in new tab

### PR DESCRIPTION
**Pull request by slawosz**
https://regulatorydelivery.atlassian.net/jira/software/projects/COSBETA/boards/24?selectedIssue=COSBETA-1430

### CODE REVIEW

File: cosmetics-web/app/helpers/responsible_persons/notifications_helper.rb [hunk 1]
```diff
@@ -197,7 +197,7 @@ def notification_summary_component_rows(component, include_shades: true, allow_e
 
   def notification_summary_label_image_link(image, responsible_person, notification)
     if image.passed_antivirus_check?
-      link_to(image.filename, url_for(image.file), class: "govuk-link govuk-link--no-visited-state")
+      link_to(image.filename, url_for(image.file), class: "govuk-link govuk-link--no-visited-state", target: "_blank", rel: "noopener")
```
Comment (scruti) [inline cosmetics-web/app/helpers/responsible_persons/notifications_helper.rb:200]: Had to Google the `noopener` bit :sweat_smile: . There is any reason we use it here but not on the formulation file links?

Comment (slawosz) [reply cosmetics-web/app/helpers/responsible_persons/notifications_helper.rb:200]: Rubocop terror ;)

Comment (scruti) [reply cosmetics-web/app/helpers/responsible_persons/notifications_helper.rb:200]: Woah that one I didn't expect!

**Review by scruti — APPROVED** [review]

**Review by nasirkhanbeis — APPROVED** [review]
looks good as expected!

**Pull request merged**